README.adoc

```diff
@@ -45,9 +45,9 @@ Embedded Derby database as a datastore for all the events.
 
 You will use JPA annotations to define an entity class whose fields are persisted to the 
 database. The interaction between your application and the database is mediated by the persistence 
-context that is managed by an EntityManager. In this guide you will learn how to use a transaction-scoped 
-container-managed persistence context EntityManager. This places the burden of managing the EntityManager's 
-lifecycle on the application server with a request scoped CDI bean. 
+context that is managed by an EntityManager. In this guide you will learn how to use a container-managed 
+EntityManager that will be obtained by the application through injection. This means, that the EntityManagers
```

Comment:
I think "how to use a container-managed EntityManager that will be obtained by the application through injection" means, "how to inject a container-managed EntityManager into the application"? 

"This means, that the EntityManagers lifecycle will be managed transparently by the Java EE container." => I think it make more sense to be, "The lifecycle of the container-managed EntityManagers will be managed by the Java EE container." But then, which type of Java EE container?

Reply:
The reason I don't like the wording of "how to inject a container-managed EntityManager into the application" is that the guide isn't ONLY teaching them how to inject a container-managed EM, it is demonstrating how to use a container-managed EM, that we obtain via injection (since EM can be injected into a component or looked up directly using JNDI).